src/components/Button/index.tsx

```diff
@@ -12,10 +12,11 @@ const Button: FC<ButtonProps> = ({
   disabled = false,
   withDownloadIcon = false,
   children,
+  isHidden = false,
```

Comment:
In general once we have props with default values we should put them at the end of the props object. Here we have a specific case, as `children` prop coming from react's `FC` interface is an optional prop - it does not make a difference, but as a good practice we can anyway move children at the top. (This is not something you introduced, but we can improve it now I think ;)) I can elaborate on the call.